extensions/positron-python/package.json

```diff
@@ -837,6 +837,12 @@
                     "description": "%python.venvPath.description%",
                     "scope": "machine",
                     "type": "string"
+                },
+                "python.installModulesInTerminal": {
+                    "default": false,
+                    "description": "%python.installModulesInTerminal.description%",
```

Comment:
Nit: Since you're using markdown styling in here, consider specifying `markdownDescription` instead so you get nicer rendering.